restrictedsessions/middleware.py

```diff
@@ -39,7 +39,11 @@ def process_request(self, request):
                 logout(request)
             else:  # logout(...) flushes the session so ensure it only happens once
                 request.session.flush()
-            logger.warning("Destroyed session due to invalid change of remote host or user agent")
+            logger.warning(
+                "Destroyed session due to invalid change of remote host or user agent",
+                user.username,
+                request.META.get('REMOTE_ADDR', None)
```

Comment:
The remote IP is not always in REMOTE_ADDR. The right address is already retrieved prior, available in the `remote_addr` local variable.